pkgs/servers/web-apps/wordpress/default.nix

```diff
@@ -9,6 +9,9 @@ stdenv.mkDerivation rec {
     sha256 = "10zjgbr96ri87r5v7860vg5ndbnjfhjhily4m9nyl60f1lbjdhrr";
   };
 
+  # exposes $WORDPRESS_CONFIG variable to configure wp-config.php path
+  patches = [ ./wordpress-wp-config.patch ];
```

Comment:
Could open a ticket here: https://core.trac.wordpress.org
and reference it back? To make a stronger case: Docker people might be happy to have feature as well.

Reply:
possible already open feature requests for that: https://core.trac.wordpress.org/ticket/41783, https://core.trac.wordpress.org/ticket/41710